composer.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "akeneo/php-api-client-ee",
+  "description": "Akeneo PIM ENTERPRISE client for the API",
+  "license": "OSL-3.0",
+  "minimum-stability": "beta",
+  "authors": [
+    {
+      "name": "Akeneo",
+      "homepage": "http://www.akeneo.com"
+    }
+  ],
+  "autoload": {
+    "psr-4": {
+      "Akeneo\\Pim\\": "src/",
```

Comment:
what about the namespaces ?

Reply:
AkeneoEnterprise if we do like in the Pim. And it allows composition of the CE client for a given class name (should not occur on a client, but maybe one day..).